proto/steward/v4/aave_v3_debt_token.proto

```diff
@@ -7,25 +7,7 @@ package steward.v4;
 
 option go_package = "/steward_proto";
 
-import "a_token.proto";
```

Comment:
Were these leftovers from when the flash loan message was in here?

Reply:
yes